assets/js/googlesitekit/data/create-settings-store.js

```diff
@@ -152,6 +154,7 @@ export const createSettingsStore = ( type, identifier, datapoint, {
 		 * @return {Object} Redux-style action.
 		 */
 		*saveSettings() {
+			const registry = yield getRegistry();
 			const values = yield registry.select( STORE_NAME ).getSettings();
```

Comment:
You're saying in the PR description that we'll need resolvers for actions that rely on selectors that have resolvers, which is the case for this one, but you didn't add a resolver for the action as far as I can tell. Am I misunderstanding something here?

Reply:
At first, we only knew about `createRegistryControl`, so the implementation required resolvers and controls for these kind of selectors. That was no longer when we realized there is also a `createRegistrySelector` (but these are the only two), so it's a result of a discovery we made in the process of the PR 😄